### PR TITLE
[openstack_edpm] Add additional paths

### PR DESCRIPTION
**Pull request by slagle**
These paths are also used by EDPM and should be collected by the plugin.

Signed-off-by: James Slagle <jslagle@redhat.com>

---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [X] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [X] Is the subject and message clear and concise?
- [X] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [X] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [X] Are any related Issues or existing PRs [properly referenced](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) via a Closes (Issue) or Resolved (PR) line?
- [X] Are all passwords or private data gathered by this PR [obfuscated](https://github.com/sosreport/sos/wiki/How-to-Write-a-Plugin#how-to-prevent-collecting-passwords)?

### CODE REVIEW

File: sos/report/plugins/openstack_edpm.py [hunk 1]
```diff
@@ -24,7 +24,10 @@ def setup(self):
         # Notes: recursion is max 2 for edpm-config
         # Those directories are present on all OpenStack nodes
         self.edpm_log_paths = [
-            '/var/lib/edpm-config/'
+            '/etc/os-net-config/'
```
Comment (arif-ali) [inline sos/report/plugins/openstack_edpm.py:27]: missing comma at the end

File: sos/report/plugins/openstack_edpm.py [hunk 2]
```diff
@@ -24,7 +24,10 @@ def setup(self):
...
+            '/var/lib/config-data/',
+            '/var/lib/edpm-config/',
+            '/var/log/containers/'
```
Comment (arif-ali) [inline sos/report/plugins/openstack_edpm.py:30]: can we add a comma at the end here too

**Review by arif-ali — CHANGES_REQUESTED** [review]

File: sos/report/plugins/openstack_edpm.py (hunk 1 above)
Comment (slagle) [reply sos/report/plugins/openstack_edpm.py]: Done.

File: sos/report/plugins/openstack_edpm.py (hunk 2 above)
Comment (slagle) [reply sos/report/plugins/openstack_edpm.py]: Done.

**Review by arif-ali — APPROVED** [review]

File: sos/report/plugins/openstack_edpm.py [hunk 3]
```diff
@@ -24,7 +24,10 @@ def setup(self):
         # Notes: recursion is max 2 for edpm-config
         # Those directories are present on all OpenStack nodes
         self.edpm_log_paths = [
-            '/var/lib/edpm-config/'
+            '/etc/os-net-config/',
+            '/var/lib/config-data/',
+            '/var/lib/edpm-config/',
+            '/var/log/containers/',
```
Comment (TurboTurtle) [inline sos/report/plugins/openstack_edpm.py:30]: Nack to `/var/log/containers` as a whole. There are _tons_ of components that drop logs there these days.

The `container_log` plugin capture this as needed, but if you need specific containers call them out explicitly in these paths, e.g. `/var/log/containers/$container/$log` which is what many of the other openstack plugins do.

Comment (slagle) [reply sos/report/plugins/openstack_edpm.py]: dropped /var/log/containers

**Review by TurboTurtle — APPROVED** [review]

**Pull request merged**